Engine/Src/Ancona/Core2D/Systems/Collision/CollisionSystem.cpp

```diff
@@ -20,18 +19,17 @@ CollisionSystem::CollisionSystem(const std::string & name, SystemManager & manag
 void CollisionSystem::OnLoad()
 {
     CreateType(NONE_COLLISION_TYPE);
-    if (FileOperations::IsFile("project/collision-types.json"))
-    {
-        auto fileStream = FileOperations::GetInputFileStream("project/collision-types.json");
-        Json::Reader reader;
-        Json::Value collisionTypesRoot;
-        reader.parse(*fileStream, collisionTypesRoot);
-        for (Json::Value & collisionType : collisionTypesRoot["collisionTypes"])
-        {
-            if (collisionType.asString() != NONE_COLLISION_TYPE)
-            {
-                CreateType(collisionType.asString());
-            }
+    auto fileStream = FileOperations::GetInputFileStream("project/collision-types.json");
+    if (!fileStream) {
+        return;
```

Comment:
you could view not having the collision types as a valid option since the "NONE" type is always created and you can manually create types (like we used to)